### ISSUE TITLE
CAH event on 11/14 - invite from social (not email) with ticketing info by Fri AM 10/30

### ISSUE THREAD

**Issue by rebeccalendl** [creativecommons/creativecommons.org#146]
Draft copy, waiting on info from CAH:

Saturday, November 14
Doors 7, Show 8
Sponsored by X (?)

Cards Against Humanity - the epic CC-licensed party game we all know and love - will be hosting an evening bash in Chicago, a full night of comedy all in support of Creative Commons! Join CAH co-creator Max Tempkin and CC CEO Ryan Merkley and X, Y, and Z at CAH’s new space (and former pickle factory!) to show your love for fine humor and your readiness to fight for a free and open web.

$50 tickets [TBD final price point]
All proceeds go directly to Creative Commons
Make your donation today

Creative Commons is a nonprofit organization at the center of a high-profile, international movement to promote sharing of creativity and knowledge. Our goal is to help realize the full potential of the Internet—universal access to research and education, full participation in culture—to drive a new era of development growth, and productivity.

**Comment by mattl** [comment]
You can change the settings for the event, including prices and address, time, date, etc here: https://donate.creativecommons.org/civicrm/event/manage/settings?reset=1&action=update&id=13

From this screen: https://donate.creativecommons.org/civicrm/event/manage?reset=1 you can click 'Event links' and then 'Registration (Live)' to get the link you need to share to get people to register for real.

CC @mmoreshead

**Comment by rebeccalendl** [comment]
Event postponed to Q1 2016
